libs/sdk-ui/src/base/react/usePromiseCache.ts

```diff
@@ -18,6 +18,14 @@ const initialState: IUsePromiseCacheState<any, any> = {
     errors: [],
 };
 
+/**
+ * @public
+ */
+export interface IIgnorePromiseCacheActions {
```

Comment:
why ...Actions? This is just some promise behavior settings, no actions included
What about

export interface IPromiseCacheSettings {
    preventResetPromises?: boolean;
    shouldCancelPromises?: boolean;
}

Reply:
I have updated this. Thank you!